site/docs/rule.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+menu: docs
+title: Rule
+permalink: /docs/rule.html
+---
+
+{: .suppress-error}
+```json
+{
+  "data": ... ,
+  "mark": "rule",
+  "encoding": ... ,
+  ...
+}
+```
+
+The `rule` mark represents each data point as a line. It can be used in two ways. First, as a line that spans the complete width or height of a view. This is useful for adding annotations. Second, a rule can be used to draw a line between two positions.
+
+
+### Usage similar to `tick`
+
+The rule mark can be used similar to a [`tick`](tick.html) mark. The difference is that a rule mark automatically spans the complete width or height of a single view such that no dimension is required. We can sue this for example to show the average price of different stocks.
+
+<span class="vl-example" data-name="rule_color_mean"></span>
+
+
+### Rules as annotations
+
+The fact that rule marks span the width or the height of a single view make them perfect for adding annotations to charts using [layering]({{site.baseurl}}/docs/layer.html).
+
+Here is an example where we use the spec from above to show the average value of stocks alongside the price curve.
+
+<span class="vl-example" data-name="layer_line_color_rule"></span>
+
+
+### Rules with two positional encodings
+
+Rules can encode two positions for x and y. This is useful to show multiple measures for a single ordinal value.
+
+For example, we can show the extent of horsepowers for cars from different locations.
```

Comment:
I think using rule as error bar is a more commonly used example.  Maybe add error bar?

Reply:
We can show how we  use aggregate with rule to show different types of error bars similar to the error bar example in Vega?

Reply:
I wanted to keep this example as simple as possible. Min and max are pretty easy to understand and I don't want to use layering here.

Reply:
I think we should keep this example, but it is important to show error bars as it is gonna be very commonly used.

Reply:
Okay, let's do that in a follow up.

Reply:
Done